### PR TITLE
KK-449 | Fix "rendered fewer hooks than expected" bug in event

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

File: src/domain/event/modal/UnenrolModal.tsx [hunk 1]
```diff
@@ -74,8 +74,8 @@ const UnenrolModal = ({
       // eslint-disable-next-line no-console
       console.error(error);
       // TODO: KK-280 Handle errors nicely
-      toast(t('registration.submitMutation.errorMessage'), {
-        type: toast.TYPE.ERROR,
+      toast.error(t('registration.submitMutation.errorMessage'), {
+        type: error,
       });
```
Comment (tuovinensanttu) [inline src/domain/event/modal/UnenrolModal.tsx:79]:
```suggestion
      toast.error(t('registration.submitMutation.errorMessage'));
```
In other files toast was used this way (without options object).

File: src/domain/profile/children/ProfileChildrenList.tsx [hunk 2]
```diff
@@ -51,8 +51,8 @@ const ProfileChildrenList: FunctionComponent = () => {
                   trackEvent({ category: 'action', action: 'Add child' });
                 })
                 .catch((error) => {
-                  toast(t('profile.addChildMutation.errorMessage'), {
-                    type: toast.TYPE.ERROR,
+                  toast.error(t('profile.addChildMutation.errorMessage'), {
+                    type: error,
                   });
```
Comment (tuovinensanttu) [inline src/domain/profile/children/ProfileChildrenList.tsx:56]:
```suggestion
                  toast.error(t('profile.addChildMutation.errorMessage'));
```
Same here.

File: src/domain/profile/modal/EditProfileModal.tsx [hunk 3]
```diff
@@ -60,8 +60,8 @@ const EditProfileModal: React.FunctionComponent<EditProfileModalProps> = ({
       trackEvent({ category: 'action', action: 'Edit profile' });
       setIsOpen(false);
     } catch (error) {
-      toast(t('registration.submitMutation.errorMessage'), {
-        type: toast.TYPE.ERROR,
+      toast.error(t('registration.submitMutation.errorMessage'), {
+        type: error,
       });
```
Comment (tuovinensanttu) [inline src/domain/profile/modal/EditProfileModal.tsx:65]:
```suggestion
      toast.error(t('registration.submitMutation.errorMessage'));
```
Same as above.

File: src/domain/registration/form/RegistrationForm.tsx [hunk 4]
```diff
@@ -123,8 +123,8 @@ const RegistrationForm: FunctionComponent = () => {
                   history.push('/registration/success');
                 })
                 .catch((error) => {
-                  toast(t('registration.submitMutation.errorMessage'), {
-                    type: toast.TYPE.ERROR,
+                  toast.error(t('registration.submitMutation.errorMessage'), {
+                    type: error,
                   });
```
Comment (tuovinensanttu) [inline src/domain/registration/form/RegistrationForm.tsx:128]:
```suggestion
                  toast.error(t('registration.submitMutation.errorMessage'));
```
Same as above.

**Comment by badrange** [comment]
Thanks for the suggestions @tuovinensanttu . 

I had actually done them on on my own computer but for some reason forgotten to add & commit & push :-D

**Pull request merged**